src/Terrain/GameMap.java

```diff
@@ -68,9 +68,34 @@ public GameMap(CommandingOfficers.Commander[] COs, MapInfo mapInfo)
           System.out.println("Warning! CO specified as owner of an uncapturable location in map " + mapInfo.mapName);
         }
       }
+
+      // Warn if the CO still doesn't have a valid HQ.
+      if( COs[co].HQLocation == null )
+      {
+        System.out.println("Warning! CO does not have any HQ assigned!");
+
+        // We still need to set HQLocation to something, so just grab the first owned property.
+        if( mapInfo.COProperties[co].length > 0 )
+        {
+          COs[co].HQLocation = mapInfo.COProperties[co][0];
```

Comment:
I don't think I like assigning a random property as the HQ. I'd rather just refuse to create a game using a malformed map.